instcheck.c

```diff
@@ -88,6 +88,18 @@ int mode;
   perm(".../",subdir,"/",file,S_IFREG,uid,gid,mode);
 }
 
+void v(home,subdir,to,from,uid,gid,mode)
```

Comment:
Sorry for planting a wrong tree for you to bark up. This is well done, but let's not use `v()` after all -- just `mv INSTALL INSTALL.md && mv SENDMAIL SENDMAIL.md` and adjust `hier.c` and `install-big.c` (while we still have it) to match.

Reply:
I appreciate your acknowledging initiating an approach that turned out to be a dead-end.  I'd rather two commits, one of which is good, than one commit which isn't.  No harm done, trying it left us a better result.  The rename feature is now pulled in favor of absolutely renaming these files.